src/libsync/discoveryphase.cpp

```diff
@@ -90,7 +90,7 @@ int DiscoveryJob::isInSelectiveSyncBlackListCallback(void *data, const QByteArra
 
 bool DiscoveryJob::checkSelectiveSyncNewFolder(const QString &path, RemotePermissions remotePerm)
 {
-    if (_syncOptions._confirmExternalStorage
+    if (_syncOptions._confirmExternalStorage && !_syncOptions._moveFilesToTrash
```

Comment:
`_moveFilesToTrash`? I don't get it

Reply:
That obviously was meant to be _newFilesAreVirtual
I guess i selected the wrong entry in auto-complete by mistake.